pkg/cloud/libvirt/client/domain_test.go

```diff
@@ -28,7 +29,7 @@ TestCases:
 	for i, tc := range testCases {
 		domainDef := libvirtxml.Domain{}
 
-		err := setCoreOSIgnition(&domainDef, tc.ignKey)
+		err := setCoreOSIgnition(&domainDef, tc.ignKey, runtime.GOARCH)
```

Comment:
why does arch needs to be passed from here? Can't `setCoreOSIgnition` access the `runtime.GOARCH` itself directly?

Reply:
The `setCoreOSIgnition` function needs the arch variable in its args as it gets it from `getHostArchitecture`. So for the test, this needs to be passed in.